### PR TITLE
Enabled Test Harness

### PR DESCRIPTION
**Pull request by r2liquibase**

### CODE REVIEW

**Comment by r2liquibase** [comment]
Thanks, Kevin! Snyk blocker resolved.

File: .github/workflows/test-harness.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: configure auth
+        id: auth
+        uses: 'google-github-actions/auth@v0'
+        with:
+          workload_identity_provider: 'projects/263903559714/locations/global/workloadIdentityPools/github-identity-pool/providers/github-identity-provider'
+          service_account: 'gh-bq-deployer@gh-integration-tests.iam.gserviceaccount.com'
```
Comment (molivasdat) [inline .github/workflows/test-harness.yml:115]: I suggest changing this to a GH secret.

File: liquibase.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+##changeLogFile: changelog.sql
+driver: com.simba.googlebigquery.jdbc.Driver
+url: jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=lolejniczakdatabricksdemo;DefaultDataset=lharness;OAuthType=0;OAuthServiceAcctEmail=sa-liquibase@lolejniczakdatabricksdemo.iam.gserviceaccount.com;OAuthPvtKeyPath=/Users/lolejniczak/Downloads/lolejniczakdatabricksdemo-51f56a38cd69.p12;
```
Comment (molivasdat) [inline liquibase.properties:3]: Use Github secrets for this

File: pom.xml [hunk 3]
```diff
@@ -55,11 +55,11 @@
             <timezone>-6</timezone>
         </developer>
     </developers>
-    
+
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <liquibase.version>0-SNAPSHOT</liquibase.version>
+        <liquibase.version>4.15.0</liquibase.version>
```
Comment (molivasdat) [inline pom.xml:62]: Latest version of Liquibase is 4.16.1

File: pom.xml [hunk 4]
```diff
@@ -94,6 +101,12 @@
             <artifactId>liquibase-core</artifactId>
             <version>${liquibase.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>1.30</version>
```
Comment (molivasdat) [inline pom.xml:107]: CVE for snakeyaml 1.30 use at least 1.31, I believe 1.32 is the latest.
https://github.com/advisories/GHSA-3mc7-4q67-w48m

File: src/main/resources/changelog.sql [hunk 5]
```diff
@@ -0,0 +1,27 @@
+--liquibase formatted sql
```
Comment (molivasdat) [inline src/main/resources/changelog.sql:1]: Remove this file from the build.  It gets included in the built extension jar file.

File: src/main/resources/liquibase.properties [hunk 6]
```diff
@@ -0,0 +1,5 @@
+##changeLogFile: changelog.sql
```
Comment (molivasdat) [inline src/main/resources/liquibase.properties:1]: Remove this file from the build.  It gets included in the built extension jar file.

**Comment by r2liquibase** [comment]
Thank you, @ewojtach !!!

**Pull request merged**